### PR TITLE
Adding a handler for the error rate

### PR DESCRIPTION
**Pull request by johandahlberg**
This adds a error rate handler.

### CODE REVIEW

File: tests/handlers/test_error_rate_handler.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        self.error_handler.qc_config = qc_config
+
+    def test_all_is_fine(self):
+        qc_config = {'name': 'ErrorHandler', 'error': 5, 'warning': 6}
+        self.set_qc_config(qc_config)
+        errors_and_warnings = list(self.error_handler.check_qc())
+        self.assertEqual(errors_and_warnings, [])
+
+    def test_warning(self):
+        qc_config = {'name': 'ErrorHandler', 'error': 3, 'warning': 5}
```
Comment (monikaBrandt) [inline tests/handlers/test_error_rate_handler.py]: The level of failure should be higher than the level of warning.

**Review by monikaBrandt — CHANGES_REQUESTED** [review]
Please, update warning and error levels in test_error_rate_handler.py

**Comment by johandahlberg** [comment]
Fixed.

**Review by monikaBrandt — APPROVED** [review]

**Comment by monikaBrandt** [comment]
Awesome! Everything looks good.

**Pull request merged**